src/sc/embedder/content/embedder.py

```diff
@@ -77,6 +78,28 @@ def EmbedderImageFieldWidget(field, request):
     return FieldWidget(field, EmbedderImageWidget(request))
 
 
+def valid_size_constraint(size):
+    """Validate if size inserted is a float number or a %.
```

Comment:
I had some issues with localization and decimal separator format, so I'll keep type `int` as you suggested for now.